login-workflow/docs/ContactSupport.md

```diff
@@ -0,0 +1,71 @@
+| hideContactSupportContent             | Prop to hide `ContactSupportContent`                      | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportIcon                | Prop to hide `ContactSupportIcon`                         | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportQuestion            | Prop to hide `contactSupportQuestion`                     | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportMessage             | Prop to hide `contactSupportMessage`                      | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportEmergency           | Prop to hide `contactSupportEmergency`                    | `boolean`                                                         | no       | `false`                                                                     |
+| hideContactSupportTechnicalAssistance | Prop to hide `contactSupportTechnicalAssistance`          | `boolean`                                                         | no       | `false`                                                                     |
+| hideActions                           | Prop to hide `Actions`                                    | `boolean`                                                         | no       | `false`                                                                     |
+| showInCard                            | Prop to wrap `ContactSupport` in `BrandedCardContainer`   | `boolean`                                                         | no       | `true`                                                                      |
+| classes                               | Style overrides                                           | `ContactSupportClasses`                                           | no       | {}                                                                          |
+| slots                                 | Prop used for each slot in `ContactSupport`               | `{cardHeader: React.ElementType; cardContent: React.ElementType}` | no       | {}                                                                                                                                     |
+| slotProps                             | Props applied to each slot                                | `{cardHeader: CardHeaderProps; cardContent: CardContentProps}`    | no       | {}                                                                                                                                     |
+
+
+</div>
+
```

Comment:
And some docs for the hidden elements should you choose to add them

### ContactSupportHiddenElements

ContactSupportHiddenElements is a prop used in the ContactSupport component to control the visibility of various elements within the component. To use ContactSupportHiddenElements, simply pass an object with the keys corresponding to the elements you want to hide set to true. If a key is not present or is set to false, the corresponding element will be visible.

#### Available Keys
The following keys can be used within the ContactSupportHiddenElements object:

`title`: Hides the title of the contact support component.
`contactSupportContent`: Hides the main content of the contact support component.
`contactSupportIcon`: Hides the contact support icon.
`contactSupportQuestion`: Hides the general questions text.
`contactSupportMessage`: Hides the contact support message.
`contactSupportEmergency`: Hides the emergency support text.
`contactSupportTechnicalAssistance`: Hides the technical assistance text.
`actions`: Hides the actions (e.g., buttons) within the contact support component.

#### Example
Here's an example of how to use the ContactSupportHiddenElements prop to hide the title and actions in the ContactSupport component:


```import { ContactSupport } from '...';

const App = () => {
  return (
    <ContactSupport
      hiddenElements={{
        title: true,
        actions: true
      }}
    />
  );
};
```
In this example, the ContactSupport component will be rendered without the title and actions (e.g., buttons). The other elements will remain visible as their corresponding keys are not set to true in the hiddenElements object.